samcli/hook_packages/terraform/copy_terraform_built_artifacts.py

```diff
@@ -268,3 +264,61 @@ def cli_exit():
     except OSError as ex:
         LOG.error("Copy/Unzip unsuccessful!", exc_info=ex)
         cli_exit()
+
+
+def create_backend_override():
+    """
+    Copies and rename the override tf file from the metadata directory to the root
+    directory of the TF application.
+    """
+    override_src_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), TF_BACKEND_OVERRIDE_FILENAME)
+    override_dest_path = os.path.join(os.getcwd(), TF_BACKEND_OVERRIDE_FILENAME + ".tf")
+    shutil.copyfile(override_src_path, override_dest_path)
```

Comment:
q: need to double check on what shutil.copyfile does on different python versions, just to ensure there are no surprises.

Reply:
It's in py27 so there should be no surprise. However, just to be consistent and not to have too many variants, I'd change to to `shutils.copy2`, which is also used in the `copytree` function in the same script